Resource.dhall

```diff
@@ -0,0 +1,54 @@
+let Resource
+    : ∀(_params : { Source : Type, Version : Type }) →
+        { Type : Type
+        , default :
+            { icon : Optional Text
+            , version : Optional _params.Version
+            , check_every : Optional Text
+            , tags : Optional (List Text)
+            , public : Optional Bool
+            , webhook_token : Optional Text
+            }
+        }
+    = λ(_params : { Source : Type, Version : Type }) →
```

Comment:
Why not just getting the `\(resource-types : Tyoe) ->'? as anyways `resource-types/*.dhall` holds all the required input.